capstone-app/src/app/modules/onboarding/onboard/onboarding.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { ContentComponent } from '../content/content.component';
+
+@Component({
+  selector: 'app-onboarding',
+  templateUrl: './onboarding.component.html',
+  styleUrls: ['./onboarding.component.scss']
+})
+export class OnboardingComponent implements OnInit {
+
+  constructor(public dialog: MatDialog) { }
+
+  openDialog() {
+    this.dialog.open(ContentComponent);
+  }
+
+  ngOnInit(): void {
+    this.openDialog();
+  }
```

Comment:
Just a style issue: If you're going to do return types for one function, do it for all